apps/gitness/src/pages/repo-list.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+  Button,
+  ListActions,
+  ListPagination,
+  Pagination,
+  PaginationContent,
+  PaginationEllipsis,
+  PaginationItem,
+  PaginationLink,
+  PaginationNext,
+  PaginationPrevious,
+  SearchBox,
+  Spacer,
+  Text
+} from '@harnessio/canary'
+import { useListReposQuery } from '@harnessio/code-service-client'
+import { PaddingListLayout, SkeletonList, RepoList, Repo } from '@harnessio/playground'
+import { Link } from 'react-router-dom'
+
+const filterOptions = [{ name: 'Filter option 1' }, { name: 'Filter option 2' }, { name: 'Filter option 3' }]
+const sortOptions = [{ name: 'Sort option 1' }, { name: 'Sort option 2' }, { name: 'Sort option 3' }]
+const viewOptions = [{ name: 'View option 1' }, { name: 'View option 2' }]
+
+const LinkComponent = ({ to, children }: { to: string; children: React.ReactNode }) => <Link to={to}>{children}</Link>
+
+export default function ReposListPage() {
+  const { isFetching, data } = useListReposQuery({ queryParams: {}, space_ref: 'test' })
```

Comment:
`space_ref` is hardcoded to `test` for now, until Jessie's PR to select spaces/projects get merged.